hack/build/version.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+
+function kubevirt::version::get_version_vars() {
```

Comment:
why _kubevirt_ prefix instead of, say, _cdi::_ ?

Reply:
this is ... inspired... by version.sh in kubevirt/kubevirt :)  Just began testing it before altering it for CDI's use.  Honestly, it seems like overkills for determining whether it's a dirty branch or not.